modules/utils.js

```diff
@@ -643,6 +643,10 @@ function mergeObjectsInGecko(origin, add, opts) {
 
     var options = opts || {},
         keepReferences = options.keepReferences;
+    
+    // Fix prototype pollution freezing the *origin* Object
+    Object.freeze(origin.__proto__);
```

Comment:
You can remove `Object.freeze(origin.__proto__);`, it's not working.

Reply:
You're right, I was experimenting on this one and I left the line that was useless :smile:

Cheers,
Mik